indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
```

Comment:
In an ideal world I'd agree with you. But you know how users are... some prefer to do things by phone/email/etc. instead of just doing it themselves in Indico itself...